code/go/0chain.net/miner/miner_test.go

```diff
@@ -209,29 +235,103 @@ func BenchmarkGenerateAndVerifyALotTransactions(b *testing.B) {
 	}
 }
 
-func SetUpSingleSelf() {
+func SetUpSingleSelf() func() {
+	// create rocksdb state dir
+	clean := setupTempRocketDBDir()
+	s, err := miniredis.Run()
+	if err != nil {
+		panic(err)
+	}
+	p, err := strconv.Atoi(s.Port())
+	if err != nil {
+		panic(err)
+	}
+	memorystore.InitDefaultPool(s.Host(), p)
+
+	memorystore.AddPool("txndb", &redis.Pool{
+		MaxIdle:   80,
+		MaxActive: 1000, // max number of connections
+		Dial: func() (redis.Conn, error) {
+			c, err := redis.Dial("tcp", s.Addr())
+			if err != nil {
+				panic(err.Error())
+			}
+			return c, err
+		},
+	})
+
+	//n1 := &node.Node{Type: node.NodeTypeMiner, Host: "", Port: 7071, Status: node.NodeStatusActive}
```

Comment:
Looks like redundant commented out code, if we don't need it better to just remove it for the sake of clarity.